internal/core/src/exec/Driver.cpp

```diff
@@ -182,23 +180,29 @@ Driver::Next(std::shared_ptr<BlockingState>& blocking_state) {
     try {                                                                      \
         call_func;                                                             \
     } catch (SegcoreError & e) {                                               \
```

Comment:
this `catch (SegcoreError & e)` branch is now identical to the `catch (std::exception & e)` branch after added `LOG_ERROR(err_msg);` to the later, this branch can be removed.